static/js/common.ts

```diff
@@ -56,26 +56,40 @@ export function adjust_mac_shortcuts(key_elem_class: string, require_cmd_style =
     const keys_map = new Map([
         ["Backspace", "Delete"],
         ["Enter", "Return"],
-        ["Home", "Fn + ←"],
-        ["End", "Fn + →"],
-        ["PgUp", "Fn + ↑"],
-        ["PgDn", "Fn + ↓"],
+        ["Home", "←"],
+        ["End", "→"],
+        ["PgUp", "↑"],
+        ["PgDn", "↓"],
         ["Ctrl", "⌘"],
     ]);
 
+    const fn_shortcuts = new Set(["Home", "End", "PgUp", "PgDn"]);
+
+    let key_elem = "code";
+
+    if (key_elem_class.slice(Math.max(0, key_elem_class.lastIndexOf(" ") + 1)) === "kbd") {
```

Comment:
Would it be better to just pass a separate parameter specifying whether it's a `kbd` or `code` element?

Reply:
(And then actually I think you get to delete this block and just have the conditional in one place, below).

Reply:
Yes. I had started with adding this as a parameter, but had been thinking of it as a string. But since we really only want `kbd` or `code`, a boolean parameter makes sense.